src/docfx/publish/PublishModelBuilder.cs

```diff
@@ -126,5 +126,10 @@ public bool TryAdd(Document file, PublishItem item)
 
             return (model, fileManifests);
         }
+
+        private bool IsDeletableFile(PublishItem item)
+        {
+            return !item.Path.StartsWith("..");
```

Comment:
Is `item.Path` always a relative path?

Reply:
yes, related to manifest file

---
In reply to: [284090685](https://github.com/dotnet/docfx/pull/4518#discussion_r284090685) [](ancestors = 284090685)

Reply:
If the output path is in another windows directory, `item.Path` will look like `D:/a/b/c`, thus this check will fail. There is a `PathUtility.Match` method that tries to do the same thing, use that method and update the implementation?